biojava-ontology/src/test/java/org/biojava/nbio/ontology/TestOboFileParsing.java

```diff
@@ -34,26 +34,52 @@
 import java.io.InputStream;
 import java.io.InputStreamReader;
 import java.util.Set;
+import java.util.Iterator;
+import org.biojava.nbio.ontology.utils.Annotation;
 
 public class TestOboFileParsing {
 
-	private static final Logger logger = LoggerFactory.getLogger(TestOboFileParsing.class);
+    private static final Logger logger = LoggerFactory.getLogger(TestOboFileParsing.class);
```

Comment:
I'm afraid all of the biojava source should use tabs (I lost that argument a while back :wink:)